pkg/awsvpc/nacl_analysis.go

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2023- IBM Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package awsvpc
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+
+	"github.com/np-guard/models/pkg/connection"
+	"github.com/np-guard/models/pkg/ipblock"
+
+	"github.com/np-guard/vpc-network-config-analyzer/pkg/commonvpc"
+)
+
+type AWSNACLAnalyzer struct {
+	naclResource       *types.NetworkAcl
+	referencedIPblocks []*ipblock.IPBlock
+}
+
+func NewAWSNACLAnalyzer(nacl *types.NetworkAcl) *AWSNACLAnalyzer {
+	res := &AWSNACLAnalyzer{naclResource: nacl}
+	return res
+}
+
+func getPortsStr(minPort, maxPort int64) string {
+	return fmt.Sprintf("%d-%d", minPort, maxPort)
+}
+
```

Comment:
We don't really need this function - can be embedded into the only `Sprintf` where it is used